worker.js

```diff
@@ -1,24 +1,19 @@
 // @flow
 const {parentPort} = require('worker_threads');
-const fs = require('fs');
+const fs = require('../lib/util/fs-normalized');
```

Comment:
this needs review to ensure we are pulling this correctly with the out-of-band copying of worker file